src/lekcja8_zadanie1_firma_szkoleniowa/Student.java

```diff
@@ -0,0 +1,13 @@
+package lekcja8_zadanie1_firma_szkoleniowa;
+
+public class Student extends Person {
+
+    public Student(String firstName, String lastName) {
+        super(firstName, lastName);
+    }
+
+    @Override
+    public String getInfo() {
+        return super.getInfo();
+    }
```

Comment:
to jest niepotrzebne. metoda `getInfo` zostanie odziedziczena

Reply:
Poprawione